src/index.js

```diff
@@ -114,28 +115,30 @@ function executePayment (_subpayments, params) {
       transfers[0].debits[0].memo = params.sourceMemo
     }
 
-    // In universal mode, each transfer executes when the last transfer in the chain
-    // has executed. The final one in the chain executes when all are prepared.
+    // In universal mode, all transfers are prepared. Then the recipient
+    // executes the transfer on the final ledger by providing a receipt. This
+    // then triggers a chain of executions back to the sender.
     //
-    // In atomic mode, all transfers execute when the notary has confirmation
-    // that all of the transfers are prepared.
-    const receiptConditionState = isAtomic ? 'prepared' : 'executed'
-    const receiptCondition = params.receiptCondition ||
-      (yield conditionUtils.getReceiptCondition(
-        transfers[transfers.length - 1],
-        receiptConditionState))
-
-    const caseID = isAtomic && (yield notaryUtils.setupCase({
+    // In atomic mode, all transfers execute when the notary receives the
+    // receipt and notifies the ledgers that it was received on time.
+    const receiptCondition = params.receiptCondition
+
+    // TODO: We could use optimistic mode if no receipt condition was specified.
```

Comment:
The reader might not know what "optimistic mode" is (I don't), so it might be good to define it here.